@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -84,6 +84,12 @@ export interface VisibleTileSetOptions {
      * Number of levels to go down when searching for fallback tiles.
      */
     quadTreeSearchDistanceDown: number;
+
+    /**
+     * Maximal number of new tiles, that can be added to the scene per frame
+     * @beta
```

Comment:
`@defaultValue` ?

Reply:
I would also mark it as `@internal` b/c it is not expected that users set that flag

Reply:
Looking at all the other properties they are not optional b/c they are resolved in `MapView`.
We should do the same here and add the optional property to `MapViewOptions`